template/android/app/src/main/java/com/helloworld/MainActivity.java

```diff
@@ -12,4 +14,9 @@
   protected String getMainComponentName() {
     return "HelloWorld";
   }
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(null);
```

Comment:
So instead of fixing it in react-native-screens you propose to make it a "standard RN development"? Moreover not all app uses react-native-screens.

https://developer.android.com/reference/android/app/Activity#onCreate(android.os.Bundle)

Reply:
Also why do it in the MainActivity? You can also do it in `ReactActivity` if it is really not needed. And no need to change template.